02_getting_started.asciidoc

```diff
@@ -9,6 +9,24 @@ The Lightning Network is accessed via software applications that can speak the L
 
 Users have the highest degree of control by running their own Bitcoin node and Lightning node. However, Lightning nodes can also use a lightweight Bitcoin client (commonly referred to as Simplified Payment Verification (SPV)) to partially validate the correctness of their blockchain.
 
+=== Lightning Explorer
+
+A Lightning network explorer is an useful tool to show the statistics of nodes, channels and network capacity.
+
+The below is an unexhausted list:
+
+* https://1ml.com/, 1ML Lightning explorer
+* https://lightblock.me/, Lightblock Lightning explorer
+* https://explorer.acinq.co/, ACINQ's Lightning explorer, with fancy visualization
+* https://www.robtex.com/lightning/node/, Robtex Lightning explorer
+
+However, using Lightning explorer **may introduce privacy issue** because, if users are careless, the website may track their IP addresses and collect their behavior records (for example, the nodes users are interested in).
+
+[TIP]
+====
+Note that Lightning explorers just illustrate information that can already be queried from the gossip protocol (that's also how they obtain those data) and those data can be easily acquired via the command line in LAPPs. So it is recommended to use commanline tools or a privacy-preserving browser (e.g., Tor browser), if a user care about their privacy. An alternative solution is that, users can build an open-source explorer (for example, using https://github.com/chemicstry/recksplorer) themselves and query data from it.
```

Comment:
Mention of LAPPs here seems out of place. Also the linked repo appears to be unmaintained as it hasn't been updated in nearly 2 years.

Reply:
@renepickhardt what do you think? should we keep  recksplorer?  (because you suggestion it in https://github.com/lnbook/lnbook/pull/147#discussion_r379919073)

Reply:
> Mention of LAPPs here seems out of place. Also the linked repo appears to be unmaintained as it hasn't been updated in nearly 2 years.

Good point! I have removed the LAPPs part.